Mon 10978 openid configuration examples (#959)
centreon/centreon-documentation: fr/administration/parameters/centreon-ui.md

- Le champ **User Information Endpoint** définit le point de terminaison des informations utilisateur, par exemple `/userinfo`.
- Le champ **End Session Endpoint** définit le point de terminaison de déconnexion, par exemple `/logout`.
- Le champ **Scope** définit la portée de l'IdP, par exemple «openid». Portée séparée par espace.

> - **Login claim value** field defines the value return form **Introspection Token Endpoint** or **User Information Endpoint**

Reviewer: ```suggestion
- The **Login claim value** field defines the value returned from **Introspection Token Endpoint** or **User Information Endpoint**
```
```suggestion
- **Login claim value** field defines the value return form **Introspection Token Endpoint** or **User Information Endpoint**
```

> - **Login claim value** field defines the value return form **Introspection Token Endpoint** or **User Information Endpoint**

Reviewer: This bit is in English

  to authenticate the user. For exmaple `sub`or `email`.
- Le champ **Login claim value** définit la variable qui renvoyée par les points de terminaison **Introspection Token Endpoint**
  ou **User Information Endpoint** pour authentifier l'utilisateur. Par exemple `sub` ou `email`.
- La case **Redirect Url** définit l'URL de redirection après connexion pour accéder à votre serveur Centreon, par exemple
  `https://192.168.0.1/centreon/index.php`.
- **Client ID** défini l'ID client.
- **Client Secret** défini le secret client.

> - La case **Use Basic Auth for Token Endpoint Authentication** oblige a utiliser la méthode `Authorization: Basic`.

Reviewer: ```suggestion
- La case **Use Basic Auth for Token Endpoint Authentication** oblige à utiliser la méthode `Authorization: Basic`.
```
```suggestion
- La case **Use Basic Auth for Token Endpoint Authentication** oblige a utiliser la méthode `Authorization: Basic`.
```

- **Disable SSL verify peer** permet de désactiver la validation des pairs SSL, ne doit être utilisé que pour le test

![image](../../assets/administration/openid-connect-configuration.png)

> Selon le fournisseur d'identité, il est nécessaire de saisir plusieurs portées (scope) afin de récupérer la valeur
> (claim) qui identifiera l'utilisateur. Ceci est indiqué dans la documentation de configuration du fournisseur.

> Il est possible de définir une URL complète pour les points de terminaison au cas où la base de l'URL est différente
> des autres.

> Il est possible de ne pas spécifier le champ **Redirect Url**, dans ce cas, le serveur Centreon enverra au

> > fournisseur de service sa propre url.

Reviewer: ```suggestion
> fournisseur de service sa propre URL.
```
```suggestion
> fournisseur de service sa propre url.
```
